java-generator/it/src/it/cert-manager/expected/Auth.expected

```diff
@@ -57,6 +59,7 @@ public class Auth implements io.fabric8.kubernetes.api.model.KubernetesResource
      */
     @com.fasterxml.jackson.annotation.JsonProperty("tokenSecretRef")
     @com.fasterxml.jackson.annotation.JsonPropertyDescription("TokenSecretRef authenticates with Vault by presenting a token.")
+    @com.fasterxml.jackson.annotation.JsonSetter(nulls=com.fasterxml.jackson.annotation.Nulls.SKIP)
```

Comment:
Before I forget, @metacosm and @manusa would you like to take a look at this encoding from the Jackson point of view and if we can rely on it in the crd-generator (eventually)?

Reply:
`Nulls` management implementation has been merged already separately, resolving.